### ISSUE TITLE
Sessionize import

### ISSUE THREAD

**Issue by martinbonnin** [HugoGresse/open-feedback#1564]
Would be nice to have sessionize import. Example json: https://sessionize.com/api/v2/72i2tw4v/view/All

**Comment by HugoGresse** [comment]
It shouldn't be very hard: 
1. Add a new setup type here: https://github.com/HugoGresse/open-feedback/tree/master/src/core/setupType 
2. Update the UI in the admin to let the user pick sessionize and set the endpoint URL and as well as the setting in the event

**Comment by anyulled** [comment]
Please assign this issue to me. I already created an adapter here:

https://github.com/dev-bcn/open-feedback-adapter/blob/main/src/app/api/2024/schedule/open-feedback-adapter.ts

**Comment by HugoGresse** [comment]
Nice!
The only input needed are a single sessionizeUrl then? 
If so, is there any quota or CORS related issue to check for?

**Comment by anyulled** [comment]
You're right, the only parameter needed is the sessionize URL that is generated for each event. We should include some instructions for proper configuration, i.e., JSON format, ungrouped results, etc.

Regarding the quota, we've been consuming the API from the www.devbcn.com website for fetching the talks, schedule, and speakers' info, and never received a 429 or similar status error.

No additional CORS configuration is required.

**Comment by HugoGresse** [comment]
Perfect! Do you think you can provide a sample on some GitHub Gist for testing purposes? 
Other than that, it should be pretty straightforward.

**Comment by anyulled** [comment]
Take a look at this one:

https://sessionize.com/api/v2/teq4asez/view/All

**Comment by HugoGresse** [comment]
I think I'll rather add a new mode here: 
![image](https://github.com/HugoGresse/open-feedback/assets/662377/e3d5aa3f-c6fa-44a7-959e-91968e4f4973)
 
Which should be clearer, even if it reuse the same stuff behind.

**Comment by anyulled** [comment]
> I think I'll rather add a new mode here: ![image](https://private-user-images.githubusercontent.com/662377/344178675-e3d5aa3f-c6fa-44a7-959e-91968e4f4973.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk1OTg2NDEsIm5iZiI6MTcxOTU5ODM0MSwicGF0aCI6Ii82NjIzNzcvMzQ0MTc4Njc1LWUzZDVhYTNmLWM2ZmEtNDRhNy05NTllLTkxOTY4ZTRmNDk3My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyOFQxODEyMjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MzQ0NDk4ZTNhNjBiMjc3MzU0ZTlhMjFiYjBlN2Y4MzEzOGZjMTJmMzZmNDFiMTQ2MmYyNzNmNzI4NzEyOTM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.0xC8H62IHDdey9be_fQtsMKyW2YMJ1kYBXo-PMtXxKA)
> 
> Which should be clearer, even if it reuse the same stuff behind.

Yes, That's what I'm working on. a new import mode with its own configuration screen.

**Comment by anyulled** [comment]
Hi @HugoGresse, I rebased my repository with the latest changes, but I'm changing some issues running the application locally. Would you mind cloning it and tell me what's missing in terms of configuration? Thanks in advance.

**Comment by HugoGresse** [comment]
Hi @anyulled sorry for the (big) delay. Do you have time for a quick debug call maybe? Loop me in at hugo.gresse@gmail.com

**Comment by anyulled** [comment]
Sure, I'll be on holidays in two weeks and able to troubleshoot together
